fs-storage/src/utils.rs

```diff
@@ -15,15 +15,8 @@ use std::path::Path;
 /// ```
 pub fn read_version_2_fs<K, V>(path: &Path) -> Result<BTreeMap<K, V>>
 where
-    K: Ord
-        + Clone
-        + serde::Serialize
-        + serde::de::DeserializeOwned
-        + std::str::FromStr,
-    V: Clone
-        + serde::Serialize
-        + serde::de::DeserializeOwned
-        + std::str::FromStr,
```

Comment:
Why do we need to remove `Clone`?

Reply:
@tareknaser isn't it good if we can loosen requirements for `V`?